core/src/main/scala/neotypes/internal/utils/travserse.scala

```diff
@@ -0,0 +1,56 @@
+package neotypes
+package internal.utils
+
+private[neotypes] object traverse {
```

Comment:
There's a type in the name of the file `travserse` :)

Reply:
Oh god, not again 😞 
Thanks for the catch 😄